axon-server-connector/src/test/java/org/axonframework/axonserver/connector/event/AxonServerEventStorageEngineIT.java

```diff
@@ -83,6 +83,12 @@ protected ProcessingContext processingContext() {
         return null;
     }
 
+    @Override
+    protected long trackingTokenPosition(int storedEvents) {
+        // AxonServer uses position = storedEvents (position 0 for empty store)
+        return storedEvents;
+    }
```

Comment:
It shows the difference between implementation - the question remains: is it expected? 
We have -1 for InMemory, 0 for Axon Server and 1 for Postgres. 
I doubt then if the FirstTrackingToken can be used with position 0.